schemas/v1/19_actor_mappings.go

```diff
@@ -0,0 +1,30 @@
+package v1
+
+func init() {
+	patches.Register(
+		19,
+		`
+CREATE TABLE IF NOT EXISTS {{ .SchemaName | default "public"}}.actor_codes (
+	cid		text  NOT NULL,
+	code	text  NOT NULL,
+	
+	PRIMARY KEY (cid, code)
+);
+COMMENT ON TABLE {{ .SchemaName | default "public"}}.actor_codes IS 'A mapping of a builtin actors CID to a human friendly name.';
+COMMENT ON COLUMN {{ .SchemaName | default "public"}}.actor_codes.cid IS 'CID of the actor from builtin actors.';
+COMMENT ON COLUMN {{ .SchemaName | default "public"}}.actor_codes.code IS 'Human-readable identifier for the actor.';
+    
+CREATE TABLE IF NOT EXISTS {{ .SchemaName | default "public"}}.actor_methods (
+	family		text  NOT NULL,
+	method_name	text  NOT NULL,
+	method  	bigint NOT NULL,
+	
+	PRIMARY KEY (family, method)
+);
+COMMENT ON TABLE {{ .SchemaName | default "public"}}.actor_methods IS 'A mapping of a builtin actors CID to a human friendly name.';
+COMMENT ON COLUMN {{ .SchemaName | default "public"}}.actor_methods.family IS 'The actor family.';
+COMMENT ON COLUMN {{ .SchemaName | default "public"}}.actor_methods.method_name IS 'Human-readable identifier for the actor method.';
+COMMENT ON COLUMN {{ .SchemaName | default "public"}}.actor_methods.method IS 'Method as bigint.';
```

Comment:
These tables should be joinable on some common field. Is that the case now?

Reply:
yes, it's joinable on the `actor_codes.cid` or the `actor_methods.method`